src/include/nfs/routines.rb

```diff
@@ -37,7 +37,7 @@ def SpecToServPath(spec)
         serv = Builtins.substring(spec, 0, path_begin)
         spec = Builtins.substring(spec, Ops.add(path_begin, 1))
       end
-      term(:couple, serv, spec)
+      Term.new(:couple, serv, spec)
```

Comment:
Because NfsRoutinesInclude is included in NfsClient::FstabEntryDialog, which has no access to the shortcut. Most likely, a final clean solution would not imply such thing. But this first step was intended to just reshuffle the code into a separate class, without changing the other existing pieces.